ecosystem/sep-0038.md

```diff
@@ -149,7 +151,8 @@ Name | Type | Description
     },
     {
       "asset": "iso4217:BRL",
-      "delivery_methods": [
+      "country_codes": ["BRA"],
+      "client_delivery_methods": [
```

Comment:
If these are always for deposit, could we use the term `deposit_delivery_methods`, and similar for anchors, `withdrawal_delivery_methods`? That builds on language we're already using for these two operations, where as `client` and `anchor` requires the reader to infer that during deposit they are the deliverer and therefore need to use this field.

cc @JakeUrban As I know you originally suggested client/anchor here.

Reply:
`deposit` and `withdraw` work too, but SEP-31 doesn't use that language. Maybe its still better than `client/anchor` since in SEP-31 the client is referred to as an anchor. I'm good with either 👍

Reply:
`send` / `receive`?

Reply:
What about `buy`/`sell`? They will be used along with parameters like buy_asset, buy_amount, sell_asset and sell_amount